akka-docs/src/main/paradox/typed/guide/tutorial_1.md

```diff
@@ -35,7 +35,7 @@ As we covered in the @scala[[Quickstart Guide](https://developer.lightbend.com/g
 
 ![actor tree diagram](diagrams/actor_top_tree.png)
 
-In fact, before your first actor is started, Akka has already created two actors in the system. The names of these built-in actors contain _guardian_. The guardian actors include:
+In fact, before your first actor is started, Akka has already created three actors in the system. The names of these built-in actors contain _guardian_. The guardian actors include:
```

Comment:
With typed, the user actor "your first" is the user guardian, so the two that are started before it is the root guardian and the system guardian, so two here is correct.

Reply:
gotcha! how about rephrasing it then like this?